src/patternfly/layouts/Page/styles.scss

```diff
@@ -1,110 +1,307 @@
+    &-nav {
+      flex: 1 0 100%;
+      order: 1;
+      // Set width value for container to scale properly
+      width: 100vw;
+      padding-left: var(--pf-l-page__header-nav--PaddingLeft);
+      // Header will may contain elements that are hidden at different breakpoints. 
+      // Using negative margins to offset header padding to allow those elements to layout naturally.
+      margin-right: calc(var(--pf-l-page__header--PaddingRight) * -1);
+      margin-left: calc(var(--pf-l-page__header--PaddingLeft) * -1);
+      overflow: hidden;
+      background-color: var(--pf-l-page__header-nav--BackgroundColor--sm);
+
+      @media screen and (min-width: $pf-global--breakpoint--lg) {
+        flex: 1;
+        order: initial;
+        width: auto;
+        padding: 0;
+        margin-right: var(--pf-l-page__header-nav--MarginRight--md);
+        margin-left: var(--pf-l-page__header-nav--MarginLeft--md);
+        background-color: transparent;
+      }
+    }
+
+    &-selector {
+      flex-shrink: 1;
+      align-items: center;
+      padding-top: var(--pf-l-page__header-selector--PaddingTop);
+      padding-bottom: var(--pf-l-page__header-selector--PaddingBottom);
+
+      @media screen and (min-width: $pf-global--breakpoint--md) {
+        padding-left: var(--pf-l-page__header-selector--PaddingLeft--md);
+      }
     }
 
     &-tools {
+      align-items: center;
       margin-left: auto; // to push it to the right
+
+      .pf-l-toolbar {
+        margin-top: var(--pf-l-page__header-tools--toolbar--MarginTop);
+        margin-bottom: var(--pf-l-page__header-tools--toolbar--MarginBottom);
+      }
     }
 
-    &-nav {
-      // this is not mobie first since we are changing the order and it'd be too much overwrite to do it with a min-width
-      @media (max-width: $pf-global--breakpoint--md) {
-        flex: 1 0 100%;
-        order: 1;
-        background-color: var(--pf-l-page__header-nav--sm--BackgroundColor);
+    &-avatar {
+      width: var(--pf-l-page__header-avatar--Height);
+      height: var(--pf-l-page__header-avatar--Width);
+
+      .pf-l-toolbar + & {
+        margin-left: var(--pf-l-page__header-avatar--MarginLeft);
       }
     }
+  }
+
+  // Sidebar
+  &__sidebar {
+    grid-area: 2 / 1;
+    z-index: var(--pf-l-page__sidebar--ZIndex);
+    width: var(--pf-l-page__sidebar--Width--sm);
+    padding-top: var(--pf-l-page__sidebar--PaddingTop);
+    padding-bottom: var(--pf-l-page__sidebar--PaddingBottom);
+    overflow: hidden;
+    background-color: var(--pf-l-page__sidebar--BackgroundColor);
+    transition: var(--pf-l-page__sidebar--Transition);
+    transform: var(--pf-l-page__sidebar--Transform--out);
+
+    @media screen and (min-width: $pf-global--breakpoint--md) {
+      flex: 1;
+      order: initial;
+      width: var(--pf-l-page__sidebar--Width--lg);
+      // Max-width value set for browser to calculate max-width while transitioning
+      max-width: var(--pf-l-page__sidebar--MaxWidth--lg);
```

Comment:
Since in this case max-width is directly attached to `--pf-l-page__sidebar--Width--lg` maybe we can just use one variable instead of 2 and add a comment.

Reply:
Updated